pwiz_tools/Skyline/TestFunctional/PasteTransitionListTest.cs

```diff
@@ -87,6 +89,13 @@ protected override void DoTest()
             // Clicking the OK button should save the column locations
             OkDialog(peptideTransitionList, peptideTransitionList.OkDialog);
 
+            // Verify that the correct columns were saved in the settings
+            var expectedColumns = new [] {protName, peptide, precursor, ignoreColumn, decoy, product, ignoreColumn, fragName};
+            // expectedColumns is the same length as targetColumns for comparison
+            var actualColumns = 
+                Settings.Default.CustomImportTransitionListColumnTypesList.GetRange(0, expectedColumns.Length);

```

Comment:
I don't get why you aren't just taking the whole Settings.Default.CustomImportTransitionListColumnTypesList for comparison?

Reply:
The last fifteen or so columns of the transition list I am inspecting here are all "ignore column" and although they are also saved in the settings, I thought comparing them might be redundant. Looking at it now though, that could be the simpler solution

Reply:
Settings.Default.CustomImportTransitionListColumnTypesList

Reply:
Yeah, I'd think that they should be the same length?